config/rbac/role.yaml

```diff
@@ -125,6 +125,32 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - appstudio.redhat.com
+  resources:
+  - spifilecontentrequests
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
I see we use only get, create and maybe list?

Reply:
would delete count since we have an owner ref?

Reply:
I dont think so, but i would test it out!

Reply:
Just verified, it doesn't count, I can remove delete.